src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/TupleExpansion.cs

```diff
@@ -94,19 +114,56 @@ private TupleExpansion(TypeAndCustomInfo typeAndInfo, int cardinality)
                 field,
                 parentFullName,
                 out fullName);
+            var name = field.Name;
+            var typeDeclaringMemberAndInfo = default(TypeAndCustomInfo);
+            var declaredTypeAndInfo = field.FieldTypeAndInfo;
+            var flags = fieldValue.EvalFlags;
+            var formatSpecifiers = Formatter.NoFormatSpecifiers;
+
+            if (field.IsRest)
+            {
+                var displayValue = fieldValue.GetValueString(inspectionContext, Formatter.NoFormatSpecifiers);
+                var displayType = ResultProvider.GetTypeName(
+                    inspectionContext,
+                    fieldValue,
+                    declaredTypeAndInfo.ClrType,
+                    declaredTypeAndInfo.Info,
+                    isPointerDereference: false);
+                var expansion = new TupleExpansion(declaredTypeAndInfo, cardinality - 7, useRawView: true);
```

Comment:
nit:  Is there any convenient way to avoid the hard-coded ```7```?  I imagine this will never change, but just in case...